src/helpers.js

```diff
@@ -117,4 +125,5 @@ export {
   setTimeoutFn as setTimeout,
   runWithRealTimers,
   checkContainerType,
+  jestFakeTimersAreEnabled,
```

Comment:
Pretty much all the changes in this file are refactors to facilitate this export.